music-quiz/src/presenters/scoreboardPopupPresenter.js

```diff
@@ -0,0 +1,52 @@
+import { Modal, Button } from "react-bootstrap";
+import React, { useState } from "react";
+import ScoreboardView from "../views/scoreboardView.js";
+
+const scores = [
+  {
+    userID: "1",
+    username: "Astra",
+    score: 10,
+  },
+  {
+    userID: "2",
+    username: "Astra2",
+    score: 9,
+  },
+  {
+    userID: "3",
+    username: "Anna",
+    score: 8,
+  },
+];
```

Comment:
Does not look like this sample data is used?

Reply:
That is correct, we forgot to remove it. We have removed it now.